models/repo.go

```diff
@@ -2043,7 +2054,7 @@ func DeleteRepository(doer *User, uid, repoID int64) error {
 	}
 
 	attachmentPaths := make([]string, 0, 20)
-	attachments := make([]*Attachment, 0, len(attachmentPaths))
+	attachments = make([]*Attachment, 0, len(attachmentPaths))
```

Comment:
maybe don't need a second `make()`?
```suggestion
	attachments = attachments[:0]
```

Reply:
@axifive @guillep2k  Thoses improvements are maybe not related to the fix and could be done outside to limit backport ?

Reply:
These are very minor corrections that don't change the logic